llvm/test/CodeGen/WebAssembly/simd-setcc-reductions.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: llc < %s -asm-verbose=false -verify-machineinstrs -disable-wasm-fallthrough-return-opt -wasm-disable-explicit-locals -wasm-keep-registers -mattr=+simd128 | FileCheck %s
```

Comment:
You probably want to use ./utils/update_llc_test_checks.py to generate these, since they're much easier to update if any of the codegen ever changes, I think simd-arith.ll has an example of what it looks like